tests/UI/pages/BO/advancedParameters/import/index.js

```diff
@@ -9,6 +9,15 @@ class Import extends BOBasePage {
 
     // Selectors
     this.downloadSampleFileLink = type => `a[href*='import/sample/download/${type}']`;
+    this.fileInputField = '#file';
+    this.nextStepButton = 'css=button >> text=Next step';
```

Comment:
```suggestion
    this.nextStepButton = 'button[name=submitImportFile]';
```

Please don't use selectors based on text (You can test the selector I suggested)